README.md

```diff
@@ -26,6 +26,16 @@ Invocation:
       -h, --help  show this help message and exit
 
 
+## Status
+
+* Basic syntax definition: finished, needs formal documentation.
+* Lexer / Tokenizer: finished.
+* Stage 1 Parsing (AST Generation): 75% finished.
+* Stage 2 Parsing (AST validation and traversal): 15% finished.
+* Stage 3 Parsing (AST optimization): not planned (for now).
+* 6502 ASM Code Generation: not yet begun.
```

Comment:
pretty sure this is no longer accurate

Reply:
why is there no react for "oops that's what I get for 2am commits"